pkgs/development/tools/sq/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, testVersion, sq }:
+buildGoModule rec {
+  pname = "sq";
+  version = "0.15.4";
+
+  src = fetchFromGitHub {
+    owner = "neilotoole";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-4CINLOHUVXQ+4e5I1fMqog6LubMm8RnbFmeuBOwALaw=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  vendorSha256 = "sha256-lNpWXKtnzwySzinNPxAKuaLqweWuS6zz8s2W4xXWlqM=";
+
+  # Some tests violates sandbox constraints.
+  doCheck = false;
+
+  ldflags = [
+    "-s" "-w" "-X github.com/neilotoole/sq/cli/buildinfo.Version=${version}"
```

Comment:
The nixpkgs manual says that the `-X` option is to make the executable aware of it's own version. But then why is the `github...` stuff needed?

Reply:
To be fair, @0x4A6F seems to be more knowledgeable about Go than me. :)

Reply:
Because It is set in [cli/buildinfo/buildinfo.go](https://github.com/neilotoole/sq/blob/957772902a7618e0d84848a4069670948f7f0746/cli/buildinfo/buildinfo.go#L9-L11).